src/index.html

```diff
@@ -11,7 +11,181 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="form-fieldset">
+        <legend>Personal information</legend>
+
+        <div class="form-field">
+          <label for="surname">Surname:</label>
+          <input
+            type="text"
+            name="surname"
+            id="surname"
+            autocomplete="off"
+            required
+          >
+        </div>
+
+        <div class="form-field">
+          <label for="name">Name:</label>
```

Comment:
If you have parent with two children you may add an empty space between them for readability (also for other)
```
<label for="name">Name: </label>

<input
  type="text"
  name="name"
  id="name"
  autocomplete="off"
>
```
Especially here when you're working with input that has a lot of attributes